group-coordinator/src/test/java/org/apache/kafka/coordinator/group/runtime/CoordinatorRuntimeTest.java

```diff
@@ -322,7 +335,7 @@ public void testScheduleLoading() {
         when(builder.build()).thenReturn(coordinator);
         when(supplier.get()).thenReturn(builder);
         CompletableFuture<CoordinatorLoader.LoadSummary> future = new CompletableFuture<>();
-        when(loader.load(TP, coordinator)).thenReturn(future);
+        when(loader.load(eq(TP), any())).thenReturn(future);
```

Comment:
i'm not sure how to go about this. The reason I had it this way was because until we call `runtime.scheduleLoadOperation()`, we don't have a context for the topic partition. but when we schedule a load operation we run `loader.load()`. If there's a way, I'll make the changes.

Reply:
I think that we could use something like:

```
when(loader.load(eq(TP), argThat(c -> c.equals(runtime.contextOrThrow(TP).coordinator)))).thenReturn(future);
```

You could push this a little further if you want:

```
private static <S extends CoordinatorShard<U>, U> ArgumentMatcher<S> coordinatorMatcher(
    CoordinatorRuntime<S, U> runtime
) {
    return c -> c.equals(runtime.contextOrThrow(TP).coordinator);
}

when(loader.load(eq(TP), argThat(coordinatorMatcher(runtime)))).thenReturn(future);
```